src/main/java/org/utplsql/api/Version.java

```diff
@@ -3,34 +3,83 @@
 import org.utplsql.api.exception.InvalidVersionException;
 
 import javax.annotation.Nullable;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toMap;
 
 /** Simple class to parse utPLSQL Version-information and provide the separate version-numbers
  *
  * @author pesse
  */
 public class Version implements Comparable<Version> {
+
+    public final static Version V3_0_0 = new Version("3.0.0", 3,0,0,null, true);
```

Comment:
I love the known versions! Maybe we could put the exact build numbers here - in addition to 3.1.3 version

```public final static Version V3_0_1 = new Version("3.0.1", 3,0,1,798, true);
    public final static Version V3_0_2 = new Version("3.0.2", 3,0,2,938, true);
    public final static Version V3_0_3 = new Version("3.0.3", 3,0,3,1266, true);
    public final static Version V3_0_4 = new Version("3.0.4", 3,0,4,1372, true);
    public final static Version V3_1_0 = new Version("3.1.0", 3,1,0,1849, true);
    public final static Version V3_1_1 = new Version("3.1.1", 3,1,1,1868, true);
    public final static Version V3_1_2 = new Version("3.1.2", 3,1,2,2134, true);
    public final static Version V3_1_3 = new Version("3.1.3", 3,1,3,2398, true);
    private final static Map<String, Version> knownVersions =
            Stream.of(V3_0_0, V3_0_1, V3_0_2, V3_0_3, V3_0_4, V3_1_0, V3_1_1, V3_1_2, V3_1_3)
```